examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -573,6 +573,98 @@
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interrupt the hanged worker process\n",
+    "\n",
+    "It is quite common in the code dealing with the multiple machine communication that some unexpected bugs or deadlock caused the worker process(es) hanged forever. Recall the worker processes are the seperated environment from the notebook. Hence, the interrupt kernel in the tool bar of notebook can interrupt the code of execution in notebook but the worker processes are still running."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's use sleep function to simulate that situation. If you run the following code and click the interrupt button afterwards, you will be able to execute other cells but the workers still continues to sleep, which will block the running of other requests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-01-16T06:36:53.518983Z",
+     "start_time": "2021-01-16T06:36:50.581376Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%px\n",
+    "import time\n",
+    "time.sleep(1000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-01-16T06:46:17.603676Z",
+     "start_time": "2021-01-16T06:46:17.545780Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%px\n",
+    "\n",
+    "# You will not see the output immediately, even if you stop the previous cell.\n",
+    "# It is because the workers are still sleeping. The interruption in notebook\n",
+    "# does not interrupt the workers.\n",
+    "print(\"Done\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Therefore, BlueFog provides a utility function `interrupt_hanged_ipengine_processes`. After running this functions, the previous sleep function will be interrupted so that you should be able to execute other functions afterwards.\n",
+    "\n",
+    "Note: `interrupt_hanged_ipengine_processes` is supported under the localhost mode only."
```

Comment:
- Try to highlight this line, and add additional comments in the code for this function.

Reply:
Done. Use * *